lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -82,7 +82,9 @@ class CategoryRenderer {
     const titleEl = this.dom.find('.lh-audit__title', auditEl);
     titleEl.appendChild(this.dom.convertMarkdownCodeSnippets(audit.result.title));
     this.dom.find('.lh-audit__description', auditEl)
-        .appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description));
+        .appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description, {
+          rating: Util.calculateRating(audit.result.score, scoreDisplayMode),
```

Comment:
@rviscomi Right now this means notApplicable will show up as "pass", do we need to separate them?

Reply:
Would be best to annotate it accordingly but not important.